[2020-06-29T14:48:34.514] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/primission/admin
request client ip:  ::ffff:127.0.0.1
request query:  {}
response time: 1593413314514
err name: TypeError
err message: Cannot read property 'forEach' of undefined
err stack: TypeError: Cannot read property 'forEach' of undefined
    at _callee$ (E:\fancy-customer\fancy-game-master\server\src\app\service\/user.js:52:18)
    at tryCatch (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:274:22)
    at Generator.prototype.<computed> [as next] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:97:21)
    at asyncGeneratorStep (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-06-29T16:27:08.393] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/primission?username=admin
request client ip:  ::1
request query:  {"username":"admin"}
response time: 1593419228392
err name: TypeError
err message: Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.
err stack: TypeError: Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.
    at _nonIterableSpread (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\nonIterableSpread.js:2:9)
    at _toConsumableArray (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\toConsumableArray.js:10:95)
    at _callee$ (E:\fancy-customer\fancy-game-master\server\src\app\service\/user.js:48:16)
    at tryCatch (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:274:22)
    at Generator.prototype.<computed> [as next] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:97:21)
    at asyncGeneratorStep (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-06-29T17:11:28.940] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/primission?username=admin
request client ip:  ::1
request query:  {"username":"admin"}
response time: 1593421888939
err name: ReferenceError
err message: _arr is not defined
err stack: ReferenceError: _arr is not defined
    at filter (E:\fancy-customer\fancy-game-master\server\src\app\service\/user.js:71:17)
    at Array.filter (<anonymous>)
    at _callee$ (E:\fancy-customer\fancy-game-master\server\src\app\service\/user.js:65:24)
    at tryCatch (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:274:22)
    at Generator.prototype.<computed> [as next] (E:\fancy-customer\fancy-game-master\server\node_modules\regenerator-runtime\runtime.js:97:21)
    at asyncGeneratorStep (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\fancy-customer\fancy-game-master\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

